pkg/volume/plugins.go

```diff
@@ -909,3 +979,57 @@ type dummyPluginProber struct{}
 
 func (*dummyPluginProber) Init() error                  { return nil }
 func (*dummyPluginProber) Probe() ([]ProbeEvent, error) { return nil, nil }
+
+type noopExpandableVolumePluginInstance struct {
```

Comment:
Can we move this new type into its own file?

Reply:
@gnufied Sure I can do that.